task/README.md

```diff
@@ -0,0 +1,56 @@
+# KubeVirt Disk Uploader Task
+
+Automates the extraction of disk and uploads it to a container registry.
```

Comment:
It would be good to put here some description why this task exists. Mention, that it can be used in combination with kubevirt tekton tasks or our example pipelines,...

Reply:
Done.